diff --git a/node_modules/formik/dist/formik.cjs.development.js b/node_modules/formik/dist/formik.cjs.development.js
index a6bc28d..9c6c3c5 100644
--- a/node_modules/formik/dist/formik.cjs.development.js
+++ b/node_modules/formik/dist/formik.cjs.development.js
@@ -915,16 +915,6 @@ function useFormik(_ref) {
     // a common gotcha in forms with both reset and submit buttons,
     // where the dev forgets to add type="button" to the reset button.
 
-
-    if ( typeof document !== 'undefined') {
-      // Safely get the active element (works with IE)
-      var activeElement = getActiveElement();
-
-      if (activeElement !== null && activeElement instanceof HTMLButtonElement) {
-        !(activeElement.attributes && activeElement.attributes.getNamedItem('type')) ?  invariant(false, 'You submitted a Formik form using a button with an unspecified `type` attribute.  Most browsers default button elements to `type="submit"`. If this is not a submit button, please add `type="button"`.')  : void 0;
-      }
-    }
-
     submitForm()["catch"](function (reason) {
       console.warn("Warning: An unhandled error was caught from submitForm()", reason);
     });
