diff --git a/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java b/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
index afe4b3e719ecfa8a950bd5fc8a0ff2ebf3e2cc1a..6c04fb7a91e5c0faf416355ee4420fe6cb7968eb 100644
--- a/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
+++ b/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
@@ -125,15 +125,11 @@ public class ImagePickerModuleImpl implements ActivityEventListener {
         boolean isPhoto = this.options.mediaType.equals(mediaTypePhoto);
         boolean isVideo = this.options.mediaType.equals(mediaTypeVideo);
 
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
-            if (isSingleSelect && (isPhoto || isVideo)) {
-                libraryIntent = new Intent(Intent.ACTION_PICK);
-            } else {
-                libraryIntent = new Intent(Intent.ACTION_GET_CONTENT);
-                libraryIntent.addCategory(Intent.CATEGORY_OPENABLE);
-            }
+        if (isSingleSelect && (isPhoto || isVideo)) {
+            libraryIntent = new Intent(Intent.ACTION_PICK);
         } else {
-            libraryIntent = new Intent(MediaStore.ACTION_PICK_IMAGES);
+            libraryIntent = new Intent(Intent.ACTION_GET_CONTENT);
+            libraryIntent.addCategory(Intent.CATEGORY_OPENABLE);
         }
 
         if (!isSingleSelect) {
