diff --git a/src/components/TextInput/TextInputOutlined.tsx b/src/components/TextInput/TextInputOutlined.tsx
index 000736b7282bddac6cc1740e168bbad07fa7f07e..3084dae9310f095443b6f6fa3aab9ea8666888d5 100644
--- a/src/components/TextInput/TextInputOutlined.tsx
+++ b/src/components/TextInput/TextInputOutlined.tsx
@@ -177,7 +177,9 @@ const TextInputOutlined = ({
 
   const pad = calculatePadding(paddingSettings);
 
-  const paddingOut = adjustPaddingOut({ ...paddingSettings, pad });
+  // Patch for multiline fields
+  const paddingOut = { paddingTop: pad, paddingBottom: pad };
+  // const paddingOut = adjustPaddingOut({ ...paddingSettings, pad });
 
   const baseLabelTranslateY = -labelHalfHeight - (topPosition + yOffset);
 
