patch-package
--- a/node_modules/apollo-client/bundle.umd.js
+++ b/node_modules/apollo-client/bundle.umd.js
@@ -831,7 +831,7 @@
             // If we have a `fetchMoreForQueryId` then we need to update the network
             // status for that query. See the branch for query initialization for more
             // explanation about this process.
-            if (typeof fetchMoreForQueryId === 'string') {
+            if (typeof fetchMoreForQueryId === 'string' && !!this.store[fetchMoreForQueryId]) {
                 this.store[fetchMoreForQueryId].networkStatus = exports.NetworkStatus.ready;
             }
         };
--- a/node_modules/apollo-client/data/queries.js
+++ b/node_modules/apollo-client/data/queries.js
@@ -93,7 +93,7 @@ var QueryStore = (function () {
         // If we have a `fetchMoreForQueryId` then we need to update the network
         // status for that query. See the branch for query initialization for more
         // explanation about this process.
-        if (typeof fetchMoreForQueryId === 'string') {
+        if (typeof fetchMoreForQueryId === 'string' && !!this.store[fetchMoreForQueryId]) {
             this.store[fetchMoreForQueryId].networkStatus = NetworkStatus.ready;
         }
     };
