diff --git a/node_modules/apollo-client/bundle.umd.js b/node_modules/apollo-client/bundle.umd.js
index f5f4159..2a69ed7 100644
--- a/node_modules/apollo-client/bundle.umd.js
+++ b/node_modules/apollo-client/bundle.umd.js
@@ -229,9 +229,10 @@
             var _this = this;
             process.env.NODE_ENV === "production" ? tsInvariant.invariant(fetchMoreOptions.updateQuery) : tsInvariant.invariant(fetchMoreOptions.updateQuery, 'updateQuery option is required. This function defines how to update the query data with the new results.');
             var combinedOptions;
+            var qid;
             return Promise.resolve()
                 .then(function () {
-                var qid = _this.queryManager.generateQueryId();
+                qid = _this.queryManager.generateQueryId();
                 if (fetchMoreOptions.query) {
                     combinedOptions = fetchMoreOptions;
                 }
@@ -248,6 +249,7 @@
                         variables: combinedOptions.variables,
                     });
                 });
+                _this.queryManager.stopQuery(qid);
                 return fetchMoreResult;
             });
         };
