diff --git a/node_modules/d3-timer/dist/d3-timer.e2e.js b/node_modules/d3-timer/dist/d3-timer.e2e.js
new file mode 100644
index 0000000..4c66447
--- /dev/null
+++ b/node_modules/d3-timer/dist/d3-timer.e2e.js
@@ -0,0 +1,13 @@
+var mockTimer = function (callback, delay, time) {
+  return {
+    stop() {},
+    restart(callbackFn, delay, time) {
+      callback(0)
+    },
+  }
+}
+exports.now = function() { return 0 };
+exports.timer = mockTimer;
+exports.timeout = mockTimer;
+exports.interval = mockTimer;
+exports.timerFlush = function () {};
