diff --git a/node_modules/react-native-sentry/android/src/main/java/io/sentry/RNSentryModule.java b/node_modules/react-native-sentry/android/src/main/java/io/sentry/RNSentryModule.java
index 13861ea..c71b97c 100644
--- a/node_modules/react-native-sentry/android/src/main/java/io/sentry/RNSentryModule.java
+++ b/node_modules/react-native-sentry/android/src/main/java/io/sentry/RNSentryModule.java
@@ -6,6 +6,7 @@ import android.content.pm.PackageManager;
 import android.support.annotation.NonNull;

 import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.JavaOnlyMap;
 import com.facebook.react.bridge.Promise;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
@@ -61,14 +62,14 @@ public class RNSentryModule extends ReactContextBaseJavaModule {
     private static AndroidEventBuilderHelper androidHelper;
     private static PackageInfo packageInfo;
     final static Logger logger = Logger.getLogger("react-native-sentry");
-    private static WritableNativeMap extra;
+    private static JavaOnlyMap extra;
     private static ReadableMap tags;
     private static SentryClient sentryClient;

     public RNSentryModule(ReactApplicationContext reactContext) {
         super(reactContext);
         this.reactContext = reactContext;
-        RNSentryModule.extra = new WritableNativeMap();
+        RNSentryModule.extra = new JavaOnlyMap();
         RNSentryModule.packageInfo = getPackageInfo(reactContext);
     }

@@ -285,7 +286,7 @@ public class RNSentryModule extends ReactContextBaseJavaModule {
     @ReactMethod
     public void clearContext() {
         Sentry.clearContext();
-        RNSentryModule.extra = new WritableNativeMap();
+        RNSentryModule.extra = new JavaOnlyMap();
         RNSentryModule.tags = null;
     }
