diff --git a/node_modules/react-native-paper/src/components/BottomNavigation.tsx b/node_modules/react-native-paper/src/components/BottomNavigation.tsx
index 3d89c97..1c64b60 100644
--- a/node_modules/react-native-paper/src/components/BottomNavigation.tsx
+++ b/node_modules/react-native-paper/src/components/BottomNavigation.tsx
@@ -9,6 +9,7 @@ import {
   Platform,
   Keyboard,
   ViewStyle,
+  EmitterSubscription,
 } from 'react-native';
 import { getBottomSpace } from 'react-native-iphone-x-helper';
 import color from 'color';
@@ -483,22 +484,31 @@ const BottomNavigation = ({
   }, []);
 
   React.useEffect(() => {
+    let showSubscription: EmitterSubscription | undefined;
+    let hideSubscription: EmitterSubscription | undefined;
     if (Platform.OS === 'ios') {
-      Keyboard.addListener('keyboardWillShow', handleKeyboardShow);
-      Keyboard.addListener('keyboardWillHide', handleKeyboardHide);
+      showSubscription = Keyboard.addListener(
+        'keyboardWillShow',
+        handleKeyboardShow
+      );
+      hideSubscription = Keyboard.addListener(
+        'keyboardWillHide',
+        handleKeyboardHide
+      );
     } else {
-      Keyboard.addListener('keyboardDidShow', handleKeyboardShow);
-      Keyboard.addListener('keyboardDidHide', handleKeyboardHide);
+      showSubscription = Keyboard.addListener(
+        'keyboardDidShow',
+        handleKeyboardShow
+      );
+      hideSubscription = Keyboard.addListener(
+        'keyboardDidHide',
+        handleKeyboardHide
+      );
     }
 
     return () => {
-      if (Platform.OS === 'ios') {
-        Keyboard.removeListener('keyboardWillShow', handleKeyboardShow);
-        Keyboard.removeListener('keyboardWillHide', handleKeyboardHide);
-      } else {
-        Keyboard.removeListener('keyboardDidShow', handleKeyboardShow);
-        Keyboard.removeListener('keyboardDidHide', handleKeyboardHide);
-      }
+      showSubscription?.remove();
+      hideSubscription?.remove();
     };
   }, [handleKeyboardHide, handleKeyboardShow]);
 
diff --git a/node_modules/react-native-paper/src/components/TextInput/TextInputOutlined.tsx b/node_modules/react-native-paper/src/components/TextInput/TextInputOutlined.tsx
index 7eb040e..120b0e1 100644
--- a/node_modules/react-native-paper/src/components/TextInput/TextInputOutlined.tsx
+++ b/node_modules/react-native-paper/src/components/TextInput/TextInputOutlined.tsx
@@ -180,7 +180,9 @@ class TextInputOutlined extends React.Component<ChildTextInputProps> {
 
     const pad = calculatePadding(paddingSettings);
 
-    const paddingOut = adjustPaddingOut({ ...paddingSettings, pad });
+    // Patch for multiline fields
+    const paddingOut = { paddingTop: pad, paddingBottom: pad };
+    // const paddingOut = adjustPaddingOut({ ...paddingSettings, pad });
 
     const baseLabelTranslateY =
       -labelHalfHeight - (topPosition + OUTLINE_MINIMIZED_LABEL_Y_OFFSET);
