diff --git a/node_modules/react-native-image-zoom-viewer/built/image-viewer.component.js b/node_modules/react-native-image-zoom-viewer/built/image-viewer.component.js
index 18c5c11..beec1d3 100644
--- a/node_modules/react-native-image-zoom-viewer/built/image-viewer.component.js
+++ b/node_modules/react-native-image-zoom-viewer/built/image-viewer.component.js
@@ -293,7 +293,7 @@ var ImageViewer = /** @class */ (function (_super) {
             imageSizes.push({
                 width: imageUrl.width || 0,
                 height: imageUrl.height || 0,
-                status: 'loading'
+                status: imageUrl.status || (imageUrl.width && imageUrl.height) ? 'success' : 'loading',
             });
         });
         this.setState({
diff --git a/node_modules/react-native-image-zoom-viewer/built/image-viewer.type.d.ts b/node_modules/react-native-image-zoom-viewer/built/image-viewer.type.d.ts
index 41b6970..2ad8db8 100644
--- a/node_modules/react-native-image-zoom-viewer/built/image-viewer.type.d.ts
+++ b/node_modules/react-native-image-zoom-viewer/built/image-viewer.type.d.ts
@@ -187,6 +187,10 @@ export interface IImageInfo {
      * 初始是否不超高 TODO:
      */
     freeWidth?: boolean;
+    /**
+     * Status, if is known in advance
+     */
+    status?: 'loading' | 'success' | 'fail';
 }
 export interface IImageSize {
     width: number;
diff --git a/node_modules/react-native-image-zoom-viewer/src/image-viewer.component.tsx b/node_modules/react-native-image-zoom-viewer/src/image-viewer.component.tsx
index 0a46b5d..8031326 100644
--- a/node_modules/react-native-image-zoom-viewer/src/image-viewer.component.tsx
+++ b/node_modules/react-native-image-zoom-viewer/src/image-viewer.component.tsx
@@ -90,7 +90,7 @@ export default class ImageViewer extends React.Component<Props, State> {
       imageSizes.push({
         width: imageUrl.width || 0,
         height: imageUrl.height || 0,
-        status: 'loading'
+        status: imageUrl.status || (imageUrl.width && imageUrl.height) ? 'success' : 'loading',
       });
     });
 
diff --git a/node_modules/react-native-image-zoom-viewer/src/image-viewer.type.ts b/node_modules/react-native-image-zoom-viewer/src/image-viewer.type.ts
index 538c943..84a2020 100644
--- a/node_modules/react-native-image-zoom-viewer/src/image-viewer.type.ts
+++ b/node_modules/react-native-image-zoom-viewer/src/image-viewer.type.ts
@@ -266,6 +266,10 @@ export interface IImageInfo {
    * 初始是否不超高 TODO:
    */
   freeWidth?: boolean;
+  /**
+   * Status, if is known in advance
+   */
+  status?: 'loading' | 'success' | 'fail';
 }
 
 export interface IImageSize {
