patch-package
--- a/node_modules/react-native-paper/src/components/Icon.js
+++ b/node_modules/react-native-paper/src/components/Icon.js
@@ -7,7 +7,7 @@ let MaterialIcons;
 
 try {
   // Optionally require vector-icons
-  MaterialIcons = require('react-native-vector-icons/MaterialIcons').default;
+  MaterialIcons = require('react-native-vector-icons/MaterialCommunityIcons').default;
 } catch (e) {
   MaterialIcons = ({ name, color, size, ...rest }) => {
     // eslint-disable-next-line no-console
