patch-package
--- a/node_modules/@types/node/index.d.ts
+++ b/node_modules/@types/node/index.d.ts
@@ -182,7 +182,6 @@ interface String {
 ************************************************/
 declare var process: NodeJS.Process;
 declare var global: NodeJS.Global;
-declare var console: Console;
 
 declare var __filename: string;
 declare var __dirname: string;
@@ -227,7 +226,7 @@ interface NodeExtensions {
     [ext: string]: (m: NodeModule, filename: string) => any;
 }
 
-declare var require: NodeRequire;
+//declare var require: NodeRequire;
 
 interface NodeModule {
     exports: any;
